pytorch_lightning/utilities/reproducibility.py

```diff
@@ -0,0 +1,21 @@
+"""Helper functions to help with reproducibility of models """
+import os
+
+import numpy as np
+import random
+import torch
+
+
+def seed_everything(seed: int = None):
+    """Function that sets seed for pseudo-random number generators  """
+    try:
+        seed = int(seed)
+    except (TypeError, ValueError):
+        return
+    random.seed(seed)
+    os.environ["PYTHONHASHSEED"] = str(seed)
+    np.random.seed(seed)
+    torch.manual_seed(seed)
```

Comment:
According to https://pytorch.org/docs/stable/notes/randomness.html#pytorch it is sufficient to use `torch.manual_seed` as this internally also seeds Cuda stuff.

Reply:
That's true, other people (mostly I saw this on kaggle) usually try to seed it all. But looking through pytorch source I see that `manual_seed` should be enough

Reply:
[manual_seed](https://pytorch.org/docs/stable/_modules/torch/random.html#manual_seed) also calls `torch.cuda.manual_seed_all(seed)`, so just `torch.manual_seed` should be fine.